test/rules/unnecessary_breaks_test.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2023, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test_reflective_loader/test_reflective_loader.dart';
+
+import '../rule_test_support.dart';
+
+main() {
+  defineReflectiveSuite(() {
+    defineReflectiveTests(UnnecessaryBreaksTest);
+  });
+}
+
+@reflectiveTest
+class UnnecessaryBreaksTest extends LintRuleTest {
```

Comment:
We should add a test for a library opted back to 2.19 to ensure it doesn't fire for older code.

Reply:
Agreed.  Done.